airbyte-cdk/python/airbyte_cdk/sources/file_based/config/parquet_format.py

```diff
@@ -0,0 +1,16 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+
+from pydantic import BaseModel, Field
+from typing_extensions import Literal
+
+
+class ParquetFormat(BaseModel):
+    filetype: Literal["parquet"] = "parquet"
+    decimal_as_float: bool = Field(
+        title="Convert Decimal Fields to Floats",
+        description="Whether to convert decimal fields to floats. This is the legacy behavior of the S3 source."
```

Comment:
nit: The context is useful, but I am a little weary of calling out a specific source in the description of a config option that is going to be potentially displayed to every source in general. maybe for migration purposes and ease we can leave it, but we should at least flag to get rid of this at some point

Reply:
removed mentions of S3 from the description and replaced it with a comment